dns-records-query-results/template.yaml

```diff
@@ -0,0 +1,63 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: Creates the necessary DNS zones and records for the query results account
+Parameters:
+  Environment:
+    Description: The environment type
+    Type: String
+    AllowedValues:
+      - build
+      - staging
+      - integration
+      - production
+
+Mappings:
+  Environment:
+    build:
+      ResultsApiDomainName: !Sub results.transaction.build.account.gov.uk
```

Comment:
I don't see anything you're subbing here?

Reply:
Whoops! Thanks, updated.